integration/token/fungible/support.go

```diff
@@ -1233,3 +1259,32 @@ func CheckPrometheusMetrics(ii *integration.Infrastructure, viewName string) {
 		Expect(v.Value).NotTo(Equal(model.SampleValue(0)))
 	}
 }
+
+func Conversion(network *integration.Infrastructure, wpm *WalletManagerProvider, user *token3.NodeReference, wallet string, typ token.Type, auditor *token3.NodeReference, issuer *token3.NodeReference, expectedErrorMsgs ...string) string {
```

Comment:
not very important but instead of passing expectedErrorMsgs I think it makes more sense to pass `errorMatchers ...GomegaMatcher`. This way you can have more complex cases that are needed already in some tests like

```
Equal("..."), Contains("..."), Not(Contains("..."))
```